engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/product-details-info.controller.js

```diff
@@ -36,7 +36,7 @@ angular.module('Bastion.products').controller('ProductDetailsInfoController',
                     }
                     return false;
                 });
-                results.unshift({id: null});
+                results.unshift({id: null, name: ''});
```

Comment:
I noticed on the product page when you try to remove a gpg_key it shows `undefined` where as the repo page just shows an empty filed, I changed the undefined to be like the repo page so we are more consistent with the UI in other places as well.